server/task-distributed/pom.xml

```diff
@@ -37,12 +37,32 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>apache-james-backends-cassandra</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
```

Comment:
it looks like we have those imports twice

Reply:
It's in the #2569 PR.
Nice catch nonetheless :)